src/router/index.ts

```diff
@@ -5,47 +5,44 @@ import SignUpView from '../views/SignUpView.vue'
 import PrivacyPolicyView from '../views/PrivacyPolicyView.vue'
 import { useUserStore } from '../stores/user'
 
+const protectedRoutes = [
+  {
+    path: '/',
+    name: 'login',
+    component: LoginView
+  },
+  {
+    path: '/sign-up',
+    name: 'sign-up',
+    component: SignUpView
+  },
+  {
+    path: '/privacy-policy',
+    name: 'privacy-policy',
+    component: PrivacyPolicyView
+  }
+]
+const publicRoutes = [
+  {
+    path: '/dashboard',
+    name: 'dashboard',
+    component: DashboardView
+  }
+]
```

Comment:
Split routes to make it clear which are public and which are not. I see now I've screwed with the naming as it's the opposite of what I've expected 🙈